server/src/index.ts

```diff
@@ -0,0 +1,49 @@
+import express from 'express';
+
+// import { ServerError } from '../src/types/types.ts';
+
+//! import middleware here:
+// import { parseUserQuery } from ' file path ';
+// import { queryAiEmbedding, aiCompletion } from ' file path ';
+// import { queryVectorDb } from ' file path ';
+
+
+const app = express();
+const PORT=3000
+
+//* this will take the plain text input and change it into JSON format
+// app.use(express.json());
+
+
+// app.post('/api', 
+//   // parseUserQuery,
+//   // queryAiEmbedding,
+//   // queryVectorDb,
+//   // aiCompletion,
+//   (req, res) => {
+//     res.locals.aiOutput = 'hey aaron';
+//   res.status(200).json({ response: res.locals.aiOutput });
+// });
+
+// //* Global error handler
+// const errorHandler: ErrorRequestHandler = (
+//   err: ServerError,
+//   _req,
+//   res,
+//   _next
+// ) => {
+//   const defaultErr: ServerError = {
+//     log: '❌ Express error handler caught unknown middleware error',
```

Comment:
@LauraSchlueter22  Please delete this dead code. This is a great thing to add to our next iteration. But, it will be less of a conflict/less confusing if we start with a clean slate.

So, delete any comments that are not needed.